browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -746,6 +768,14 @@ void RewardsDOMHandler::RestorePublishers(const base::ListValue *args) {
   }
 }
 
+void RewardsDOMHandler::RestorePublisher(const base::ListValue *args) {
+  if (rewards_service_) {
+    std::string publisherKey;
+    args->GetString(0, &publisherKey);
```

Comment:
please use new approach for getting this value:
`CHECK_EQ(1U, args->GetSize());` (at the top of the function)
`args->GetList()[0].GetString()`

Reply:
done